apps/src/lab2/views/dialogs/GenericDropdown.tsx

```diff
@@ -49,10 +52,13 @@ const GenericDropdown: React.FunctionComponent<GenericDropdownProps> = ({
   title,
```

Comment:
noting a clean up to do at some point: on (new) line 43 there is an unnecessary console log

Reply:
Removed!